lang/welsh.txt

```diff
@@ -0,0 +1 @@
+##plural 0
```

Comment:
Are all these languages actually `plural 0`? Or is it just too much copy/paste?

Reply:
I have no idea, I didn't realise the values were meaningful in the original stubs (I assumed they'd be overwritten by EINTS).

I'll check them against the OpenTTD versions. :)